shared-module/displayio/Display.c

```diff
@@ -136,12 +136,15 @@ void common_hal_displayio_display_construct(displayio_display_obj_t *self,
 
     // Set the group after initialization otherwise we may send pixels while we delay in
     // initialization.
-    common_hal_displayio_display_show(self, &circuitpython_splash);
+    common_hal_displayio_display_set_root_group(self, &circuitpython_splash);
     common_hal_displayio_display_set_auto_refresh(self, auto_refresh);
 }
 
 bool common_hal_displayio_display_show(displayio_display_obj_t *self, displayio_group_t *root_group) {
-    return displayio_display_core_show(&self->core, root_group);
+    if (root_group == NULL) {
+        return displayio_display_core_set_root_group(&self->core, &circuitpython_splash);
```

Comment:
```suggestion
        root_group = &circuitpython_splash;
```

Reply:
Thank you. Latest commit changes to this and adds the same handling to framebuffer and epaper displays.